src/DistributedLock.Tests/Tests/Etcd/EtcdLockTest.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Medallion.Threading.Etcd;
+using NUnit.Framework;
+
+namespace Medallion.Threading.Tests.Etcd;
+
+[Category("CI")]
+public class EtcdLockTest
+{
+    private readonly EtcdClusterSetup _etcdClusterBuilder = EtcdSetupFixture.EtcdClusterSetup;
+
+
+    [Test]
+    public void EtcdBasicAcquireLockSync_HappyPath()
+    {
+        using var client = this._etcdClusterBuilder.CreateClientToEtcdCluster();
+        var lock2 = new EtcdLeaseDistributedLock(client, "etcd");
+        using var handle2 = lock2.TryAcquire();
+        Assert.That(handle2, Is.Not.Null, "Failed to acquire lock");
+    }
+
+    [Test]
+    public async Task EtcdBasicAcquireLockAsync_HappyPath()
+    {
+        using var client = this._etcdClusterBuilder.CreateClientToEtcdCluster();
+        var lock2 = new EtcdLeaseDistributedLock(client, "etcd");
+        await using var handle2 = await lock2.TryAcquireAsync();
+        Assert.That(handle2, Is.Not.Null, "Failed to acquire lock");
+    }
```

Comment:
I don't think these 3 test cases add anything that isn't covered by the combinatorial cases. Instead, use this class to test for any Etcd-specific behavior.

Notably, we should test all constructor argument validations here, including for the options classes!